gradio/flagging.py

```diff
@@ -51,9 +52,9 @@ def _get_dataset_features_info(is_new, components):
                 headers.append(component.label + " file")
                 for _component, _type in file_preview_types.items():
                     if isinstance(component, _component):
-                        infos["flagged"]["features"][component.label + " file"] = {
-                            "_type": _type
-                        }
+                        infos["flagged"]["features"][
+                            component.label or "" + " file"
```

Comment:
`(component.label or "") + " file"` 
not sure if order of operations does that already but better to be explicit

Reply:
Sg thanks